filesystems/blob/src/test/scala/cromwell/filesystems/blob/BlobPathBuilderSpec.scala

```diff
@@ -55,8 +55,7 @@ class BlobPathBuilderSpec extends AnyFlatSpec with Matchers{
     blobPath.endpoint should equal(endpoint)
     blobPath.pathAsString should equal(testString)
     blobPath.pathWithoutScheme should equal(endpointHost + "/" + store + evalPath)
-
-    val is = Files.newInputStream(blobPath.nioPath)
+    val is = blobPath.newInputStream()
```

Comment:
Does this new way of getting the input stream use `blobPath.nioPath` under the covers?

Reply:
indeed, I think both ways work now, but I left it using this new way just since it was neater